app/modules/kit-library/assets/js/components/kit-already-applied-dialog.js

```diff
@@ -0,0 +1,26 @@
+import { Dialog } from '@elementor/app-ui';
+
+export default function KitAlreadyAppliedDialog( props ) {
+	return (
+		<Dialog
+			title={ __( 'You\'ve already applied a Kit.', 'elementor' ) }
+			text={ <>
+				{ __( 'Applying two Kits on the same website will mix global styles and colors and hurt your site\'s performance.', 'elementor' ) }
+				<br /><br />
+				{ __( 'Remove the existing Kit before applying a new one.', 'elementor' ) }
+			</> }
+			approveButtonText={ __( 'Remove existing Kit', 'elementor' ) }
+			approveButtonColor="primary"
+			approveButtonOnClick={ () => location.href = elementorAppConfig.admin_url + 'admin.php?page=elementor-tools&referrer_kit=' + props.kitId + '#tab-import-export-kit' }
```

Comment:
I don't think hardcoded link is the right way to do it.
Maybe think of a different solution? Maybe externalize the tab link from the import/export tools tab.